test/functional/mempool_wtxid.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""
+Test mempool acceptance in case of an already known transaction
+with identical non-witness data different witness.
+"""
```

Comment:
nit: I think the test could be called "mempool_wtxid_acceptance" because this is what it does exercising our mempool logic w.r.t to accepting candidates identified by wtxid imo ?

Reply:
Done!